spec/views/post_index_spec.rb

```diff
@@ -48,20 +48,14 @@
     expect(page).to have_content("Likes: #{@post1.likes_count}")
   end
 
-  scenario 'display Add new post button' do
-    visit user_posts_path(@user1)
-    expect(page).to have_button('Add new post')
-  end
-
   scenario 'clicking on a post redirects to the post show page' do
     visit user_posts_path(@user1)
     click_link @user1.posts[0].title
     expect(current_path).to eq(user_post_path(@user1, @user1.posts[0]))
   end
 
-  scenario 'clicking on a Add new post redirects to the post new page' do
+  scenario 'display pagination button' do
     visit user_posts_path(@user1)
-    click_link('Add new post')
-    expect(current_path).to eq(new_user_post_path(@user1))
+    expect(page).to have_button('Pagination')
   end
```

Comment:
- [ ] Dear @binyamolango, I see you have made changes to your test as suggested by the previous reviewer. But you have changed a button name and test the button with a name `pagination`, while in reality what we want to test is `Tests that asserts you can see a section for pagination if there are more posts than fit on the view.`. we still ned the `add new post` button, yet we wanna assert if the page cannot hold all the poste's it should show a paggenation.